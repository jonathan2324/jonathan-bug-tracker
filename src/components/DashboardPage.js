import React from 'react';
import BugList from './BugList'
import BugListFilters from '../components/BugListFilters'



const DashboardPage = () => {
  return (
    <div >
        <BugListFilters />
        <BugList />
    </div>
  );
}

export default DashboardPage;
