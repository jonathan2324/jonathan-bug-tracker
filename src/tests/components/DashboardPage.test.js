import React from 'react'
import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'
import DashboardPage from '../../components/DashboardPage'



test('Render snapshot of dashboard page', () => {
    const wrapper = shallow(<DashboardPage />)
    expect(toJSON(wrapper)).toMatchSnapshot()
})