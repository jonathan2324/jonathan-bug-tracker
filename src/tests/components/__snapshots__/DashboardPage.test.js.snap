// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render snapshot of dashboard page 1`] = `
<div>
  <React.Memo />
  <React.Memo />
  <React.Memo />
</div>
`;
